py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -56,14 +56,14 @@ def one_user_validator(node: Node) -> bool:
 
 @dynamo_tensorrt_converter(
     torch.ops.aten.native_batch_norm.default, capability_validator=one_user_validator
-)
-@dynamo_tensorrt_converter(torch.ops.aten.batch_norm.default)
-@dynamo_tensorrt_converter(torch.ops.aten.batch_norm)
+)  # type: ignore[misc]
+@dynamo_tensorrt_converter(torch.ops.aten.batch_norm.default)  # type: ignore[misc]
+@dynamo_tensorrt_converter(torch.ops.aten.batch_norm)  # type: ignore[misc]
```

Comment:
Based on what George said, Naren had removed all `# type: ignore[misc]` in a recent refactoring PR, so we can just use `--no-verify` while committing to bypass those complaints.

Reply:
Cool